### PR TITLE
Hire developer screen done

### PR DESCRIPTION
**Pull request by kalash-qbatch**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const strapiConfig = {
   collectionTypes: [
     'contact',
     'book-meeting',
+    'requirement',
```
Comment (ateeq-rehman-qbatch) [inline gatsby-config.js:26]: why are you using requirement 3 times?

File: src/pages/hire-developers/four-steps-process.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Container from "../../components/UiComponent/Container";
+import CountNum from '../../../static/arrow-line.svg'
+import { FourStepsProcessWrapper } from "./style";
+
+const FourStepsProcess = () => {
+  const FourStepsProcessData = [
+    {
+      img: CountNum,
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/four-steps-process.js:9]: you can directly access image from static folder like "/arrow-line.svg" no need to import and add this json in constant folder and import here

File: src/pages/hire-developers/hiring-hassle.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import StopWatch from "../../../static/stopwatch.svg";
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/hiring-hassle.js:2]: no need to import from static folder you can directly access image like "/stopwatch.svg" in src

File: src/pages/hire-developers/hiring-hassle.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import Retention from "../../../static/retention-01.svg";
+import FastTime from "../../../static/fast-time.svg";
+import Cost from "../../../static/cost-01.svg";
+import Hours from "../../../static/dev-match-time-1.svg";
+import Container from "../../components/UiComponent/Container";
+
+import { HiringHassleWrapper } from "./style";
+
+const HiringHassle = () => {
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/hiring-hassle.js:11]: add this json constant file and import here

File: src/pages/hire-developers/reason-to-hire.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import Container from "../../components/UiComponent/Container";
+import Button from "../../components/UiComponent/Button";
+import CheckIcon from "../../../static/checked-mark.svg";
+import { ReasonToHireWrapper } from "./style";
+
+const ReasonToHire = () => {
+  const cardsData = [
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/reason-to-hire.js:8]: add this json constant file and import here

File: src/pages/hire-developers/remote-developer.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Developer from "../../../static/hiring-senior-devs-qa-engineers.svg";
+import ScaleImage from "../../../static/scaling-team-inescapable-hassle.svg";
+import GrouthImage from "../../../static/different-timezone-hindering-growth.svg";
+import Product from "../../../static/ruining-product-health.svg";
+import Container from "../../components/UiComponent/Container";
+import { RemoteDeveloperStyleWrapper } from "./style";
+
+const RemoteDeveloper = () => {
+  const RemoteDeveloperData = [
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/remote-developer.js:10]: add this json constant file and import here

File: src/pages/hire-developers/share-your-requirements.js [hunk 7]
```diff
@@ -0,0 +1,408 @@
+import React, { useState, useEffect } from "react";
+import Container from "../../components/UiComponent/Container";
+import BarIcon from "../../../static/bar-icon.svg";
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/share-your-requirements.js:3]: please directly access static images

File: src/pages/hire-developers/share-your-requirements.js [hunk 8]
```diff
@@ -0,0 +1,408 @@
...
+import { ShareYourRequirementsWrapper } from "./style";
+import Input from "../../components/UiComponent/Input";
+
+const ShareYourRequirements = () => {
+  const [className, setClassName] = useState(null);
+  const [step, setStep] = useState(0);
+  const [stepperText, setStepperText] = useState([]);
+  const [userDetails, setUserDetails] = useState('');
+  const [error, setError] = useState(false);
+  const data = [
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/share-your-requirements.js:21]: add this json constant file and import here

File: src/pages/hire-developers/software-developers.js [hunk 9]
```diff
@@ -0,0 +1,253 @@
+import React from "react";
+
+import { CgScrollH } from "react-icons/cg";
+
+import Container from "../../components/UiComponent/Container";
+import Qbatch from "../../../static/logo-qbatch.svg";
+import { SoftwareDevelopersWrapper } from "./style";
+
+const SoftwareDevelopers = () => {
+  const dataColumn1 = [
```
Comment (ateeq-rehman-qbatch) [inline src/pages/hire-developers/software-developers.js:10]: add this json constant file and import here

File: src/pages/hire-developers/affordable-hiring.js [hunk 10]
```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Container from "../../components/UiComponent/Container";
+import Button from "../../components/UiComponent/Button";
+import { AffordableHiringWrapper } from "./style";
+
+const AffordableHiring = () => {
+  const AffordableHiringData = [
```
[the remark this answers is not in the document]
Comment (kalash-qbatch) [reply src/pages/hire-developers/affordable-hiring.js:7]: done

File: src/pages/hire-developers/hiring-hassle.js (hunk 3 above)
Comment (kalash-qbatch) [reply src/pages/hire-developers/hiring-hassle.js:2]: done

File: src/pages/hire-developers/hiring-hassle.js (hunk 4 above)
Comment (kalash-qbatch) [reply src/pages/hire-developers/hiring-hassle.js:11]: done

File: src/pages/hire-developers/perfect-dev.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+import Container from "../../components/UiComponent/Container";
+import Dev from "../../../static/devs-img.png";
+import Devs from "../../../static/right-img.svg";
+import Devss from "../../../static/left-img.svg";
+import Button from "../../components/UiComponent/Button";
+import { PerfectDevStyleWrapper } from "./style";
+
+const PerfectDev = () => {
+
+  const DevData = [
```
[the remark this answers is not in the document]
Comment (kalash-qbatch) [reply src/pages/hire-developers/perfect-dev.js:11]: done

**Comment by kalash-qbatch** [comment]
changes done

**Pull request merged**